db/migrate_cards/20151006130554_initiative_participant.rb

```diff
@@ -0,0 +1,23 @@
+# -*- encoding : utf-8 -*-
+
+class InitiativeParticipant < Card::Migration
+  def up
+    cardname = 'Initiative+Participants+*type plus right+*structure'
+    participant_search_card = Card.fetch cardname
+    participant_search_card.delete if participant_search_card
+
+    card = Card['Participant']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.